tests/core/test_appdir.cpp

```diff
@@ -121,6 +121,29 @@ namespace AppDirTest {
 
         ASSERT_TRUE(is_regular_file(destination));
     }
+
+    TEST_F(AppDirUnitTestsFixture, symlinkFile) {
+        auto destination = tmpAppDir / "usr/share/doc/simple_application/test123";
+        appDir.deployFile(SIMPLE_FILE_PATH, destination);
+        appDir.executeDeferredOperations();
+
+        ASSERT_TRUE(is_regular_file(destination));
+
+
+        appDir.symlinkFile(destination,tmpAppDir / "relative_link", true);
+
+        auto res = read_symlink(tmpAppDir / "relative_link");
+        auto expected = relative(destination, tmpAppDir);
+        ASSERT_TRUE(res == expected);
+
+
+//        appDir.symlinkFile(destination,tmpAppDir / "absolute_link", false);
```

Comment:
Why is this code commented? Please remove it, or add a comment describing the good reason for this.

Reply:
Is because of this https://github.com/linuxdeploy/linuxdeploy/blob/542edaf410a191aeb973c38cd9fd48b84ac0f530/src/core/appdir.cpp#L111

Reply:
fixded typo.

Reply:
Seems we should add two in-source comments there that this isn't supported yet (`TODO: ...`)?